ci/templates/ci/dashboard.html

```diff
@@ -180,6 +198,30 @@ <h2 class="dashhead-title">{{ dashboard.name }} Dashboard</h2>
       : [widget];
   }
 
+  function timeframeChange(selected) {
+    var start = '-1d';
+    switch (selected.value) {
+      case 'last-week':
+        start = '-7d';
+        break;
```

Comment:
We probably want to set the resolution (`interval`) for each of these to a reasonable value for the time frame.